src/lib/edge-functions/proxy.js

```diff
@@ -175,6 +182,9 @@ const prepareServer = async ({
       inspectSettings,
       port,
     })
+
+    log(`${NETLIFYDEVLOG} Edge function server running`)
```

Comment:
I'm not sure how useful this information will be to users, and I don't think we should be adding user-facing logs just to serve our tests. How do you feel about only printing this if `debug` mode is on (and adjusting the tests so that they use that flag)? Alternatively, we could print this if `process.env.NODE_ENV === 'test'`?

Reply:
That makes sense